simphony/cuds/lattice.py

```diff
@@ -25,30 +24,33 @@ def __init__(self, index, data=None):
             self.data = DataContainer(data)
 
 
-class Lattice(ABCLattice):
-    """
-    A Bravais lattice;
-    stores references to data containers (node related data).
+class MetaLattice(ABCLattice):
```

Comment:
**Naming**: I would avoid of the `Meta` in class names unless they are actually defining a python meta-class. I think probably a `BaseLattice` is a better name for the common ancestor of Lattice and FileLattice